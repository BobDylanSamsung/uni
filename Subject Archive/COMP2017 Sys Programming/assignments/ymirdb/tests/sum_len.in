set a 1 2 3
set b 1 2 -3 4 -5
set c 20
sum a
sum b
sum c
len a
len b
len c
append a 4 7 9
push b 1 -10 4
pop c
list entries
sum a
sum b
sum c
len a
len b
len c
bye
