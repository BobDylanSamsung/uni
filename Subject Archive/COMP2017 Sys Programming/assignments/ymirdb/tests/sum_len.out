ok

ok

ok

6

-1

20

3

5

1

c []
b [4 -10 1 1 2 -3 4 -5]
a [1 2 3 4 7 9]

26

-6

0

6

8

0

bye
